src/components/navbar/togglemenu/ToggleMenu.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Btn} from '../NavBarStyles'
+
+
+const ToggleMenu = () => {
+    return ( 
+        <div>
+        <Btn>
```

Comment:
Not really important. But `Btn` can be renamed to `BtnContent` or `BtnWrapper`

Next time then :)

Reply:
ok ;)